source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_list_collector.jsonnet

```diff
@@ -75,6 +75,43 @@ local editQuestion(questionTitle) = {
           ],
         },
         instruction: 'Tell the respondent to turn to <strong>Showcard 2</strong> or show them the Electronic Showcard below',
+        definitions: [
+          {
+            title: 'Electronic Showcard',
+            contents: [
+              {
+                description: 'Include',
+              },
+              {
+                list: [
+                  {
+                    text: 'family members (including partners, children and babies born on or before {census_date} even if still in hospital)',
+                    placeholders: [
+                      placeholders.censusDate,
+                    ],
+                  },
+                  'students and schoolchildren who lived away from home during term time ',
+                  'housemates, tenants or lodgers',
+                  {
+                    text: 'people who usually lived here on {census_date} but have since moved out',
+                    placeholders: [
+                      placeholders.censusDate,
+                    ],
+                  },
+                ],
+              },
+              {
+                description: 'Include people temporarily staying on such as',
+              },
+              {
+                list: [
+                  'people staying temporarily who did not have another UK address, for example, UK residents between addresses or currently without a home',
+                  'people from outside the UK who were staying in the UK for 3 months or more',
+                ],
+              },
```

Comment:
I can't see this content on the spreadsheet for `anyone-else-list-collector`.

Reply:
Sorry, that got messed up when I had to merge master with couple of conflicting changes, it's a remnant of "Include Panel"